<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MuokattavaRaportti</class>
 <widget class="QWidget" name="MuokattavaRaportti">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="pvmKehys">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <widget class="QDateEdit" name="alkaa2Date">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QDateEdit" name="alkaa3Date">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Sarake 1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QDateEdit" name="loppuu2Date">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="alkaa1Date">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="paattyyLabel">
        <property name="text">
         <string>Päättyy</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="sarake2Box">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Sarake 2</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateEdit" name="loppuu1Date">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QDateEdit" name="loppuu3Date">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="alkaaLabel">
        <property name="text">
         <string>Alkaa</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="sarake3Box">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Sarake 3</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="tyyppi1"/>
      </item>
      <item row="3" column="2">
       <widget class="QComboBox" name="tyyppi2"/>
      </item>
      <item row="3" column="3">
       <widget class="QComboBox" name="tyyppi3"/>
      </item>
      <item row="0" column="4">
       <widget class="QCheckBox" name="sarake4Box">
        <property name="text">
         <string>Sarake 4</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QDateEdit" name="alkaa4Date">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QDateEdit" name="loppuu4Date">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QComboBox" name="tyyppi4"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="muotoLabel">
       <property name="text">
        <string>Raportin muoto</string>
       </property>
       <property name="buddy">
        <cstring>muotoCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="muotoCombo"/>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="kohdennusCheck">
       <property name="text">
        <string>Kohdennuksella</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/kohdennus.png</normaloff>:/pic/kohdennus.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="KohdennusCombo" name="kohdennusCombo">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Kieli</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="kieliCombo"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="erittelyCheck">
     <property name="text">
      <string>Tulosta tilikohtaiset erittelyt</string>
     </property>
     <property name="icon">
      <iconset resource="../pic/pic.qrc">
       <normaloff>:/pic/format-list-unordered.png</normaloff>:/pic/format-list-unordered.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KohdennusCombo</class>
   <extends>QComboBox</extends>
   <header location="global">tools/kohdennuscombo.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../pic/pic.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>sarake2Box</sender>
   <signal>toggled(bool)</signal>
   <receiver>alkaa2Date</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sarake2Box</sender>
   <signal>toggled(bool)</signal>
   <receiver>loppuu2Date</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sarake3Box</sender>
   <signal>toggled(bool)</signal>
   <receiver>alkaa3Date</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>605</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>605</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sarake3Box</sender>
   <signal>toggled(bool)</signal>
   <receiver>loppuu3Date</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>605</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>605</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kohdennusCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>kohdennusCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>642</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sarake4Box</sender>
   <signal>toggled(bool)</signal>
   <receiver>alkaa4Date</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>703</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>696</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sarake4Box</sender>
   <signal>toggled(bool)</signal>
   <receiver>loppuu4Date</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>671</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>661</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sarake2Box</sender>
   <signal>toggled(bool)</signal>
   <receiver>tyyppi2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sarake3Box</sender>
   <signal>toggled(bool)</signal>
   <receiver>tyyppi3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>545</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>540</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sarake4Box</sender>
   <signal>toggled(bool)</signal>
   <receiver>tyyppi4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>733</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>733</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
